README.md

```diff
@@ -1 +1,47 @@
-Coming soon
+# PostgreSQL Kubernetes Operator
+
+## Overview
+
+This repository hosts the PostgreSQL Kubernetes Operator.
+
+## Description
+
+The PostgreSQL Kubernetes Operator deploys and operates the [PostgreSQL](https://www.postgresql.org/about/) database on Kubernetes clusters.
+
+This operator provides a Postgres database with replication enabled (one master instance and one or more hot standby replicas). The Operator in this repository is a Python script which wraps the LTS Postgres versions distributed by [Ubuntu](https://hub.docker.com/r/ubuntu/postgres), providing lifecycle management and handling events (install, configure, integrate, remove).
+
+## Usage
+
+In order to install this charm, execute the following commands (replace `SECRET_PASSWORD` with a strong password):
+
+```bash
+# Clone this charm respository
+$ git clone https://github.com/canonical/postgresql-k8s-operator
+$ cd postgresql-k8s-operator
+# Build the charm
+$ charmcraft pack
+# Create a model for our deployment
+$ juju add-model postgres
```

Comment:
Once this is published, I'd generally move the manual deployment instructions into `CONTRIBUTING.md` - see https://github.com/jnsgruk/kubernetes-dashboard-operator/blob/main/CONTRIBUTING.md

The README should be more user focused than developer focused :)

Reply:
Good advice. I will move these instructions in a new PR before publishing the package.